vllm/spec_decode/draft_model_runner.py

```diff
@@ -327,7 +364,7 @@ def execute_model(
                         count += 1
 
             # Prepare inputs for the next step
-            if step != num_steps - 1:
+            if step != num_steps - 1 and not self.mtp:
```

Comment:
Is the multi-step logic omitted here, and `self.mtp` is just using `TP1DraftModelRunner` in `is_fallback` mode?